src/test/java/teammates/test/pageobjects/InstructorFeedbacksPage.java

```diff
@@ -253,7 +253,13 @@ public void copyFeedbackSession(String feedbackSessionName, String courseId) {
         
         selectDropdownByVisibleValue(copiedCourseIdDropdown, courseId);
         
-        clickCopyTableAtRow(0);
+        clickCopyTableRadioButtonAtRow(3);
```

Comment:
This doesn't actually test whether or not the functionality is broken.

On master ->

![screen shot 2015-06-18 at 8 24 08 pm](https://cloud.githubusercontent.com/assets/6291947/8231016/133f2aac-15f8-11e5-8748-79b4b863eafd.png)
![screen shot 2015-06-18 at 8 24 02 pm](https://cloud.githubusercontent.com/assets/6291947/8231017/1340efd6-15f8-11e5-9e89-732a5cd08df4.png)
![screen shot 2015-06-18 at 8 23 49 pm](https://cloud.githubusercontent.com/assets/6291947/8231018/13420af6-15f8-11e5-989f-ff1719f289f1.png)
![screen shot 2015-06-18 at 8 23 35 pm](https://cloud.githubusercontent.com/assets/6291947/8231019/13447714-15f8-11e5-8b40-42b442c20478.png)
![screen shot 2015-06-18 at 8 23 30 pm](https://cloud.githubusercontent.com/assets/6291947/8231020/1345c4ca-15f8-11e5-83b9-fe5f1384f4a9.png)
![screen shot 2015-06-18 at 8 23 21 pm](https://cloud.githubusercontent.com/assets/6291947/8231021/134ae068-15f8-11e5-8db8-b95e00435c24.png)
![screen shot 2015-06-18 at 8 23 13 pm](https://cloud.githubusercontent.com/assets/6291947/8231022/1368161a-15f8-11e5-8e43-e33cdf64cc37.png)